packages/SwingSet/test/device-mailbox/device-mailbox.test.js

```diff
@@ -68,9 +71,13 @@ test('mailbox outbound', async t => {
     },
   });
 
-  const s2 = buildMailboxStateMap();
-  s2.populateFromData(s.exportToData());
-  t.deepEqual(s.exportToData(), s2.exportToData());
+  const mailboxStorageCopy = makeEphemeralMailboxStorage(
+    exportMailboxData(mailboxStorage),
+  );
+  t.deepEqual(
+    exportMailboxData(mailboxStorageCopy),
+    exportMailboxData(mailboxStorage),
+  );
```

Comment:
Nit: why not the following?

```suggestion
  const mailboxDataExport = exportMailboxData(mailboxStorage);
  const mailboxCopyDataExport = exportMailboxData(
    makeEphemeralMailboxStorage(mailboxDataExport),
  );
  t.deepEqual(mailboxCopyDataExport, mailboxDataExport);
```